src/controllers/auth.controller.ts

```diff
@@ -8,28 +9,28 @@ import {
   Res
 } from 'routing-controllers';
 import { getRepository } from 'typeorm';
-import * as _ from 'lodash';
 import { User } from '@entities/user.entity';
 import { createJWT } from '@services/jwt.service';
 
 @JsonController('/auth')
 export class AuthController {
   @Post('/signup')
-  async signUp(@Body({ validate: false }) user: User, @Res() response: any) {
+  async signUp(@Body({ validate: false }) user: User) {
     let newUser;
```

Comment:
This can be also declared on the signUp method: `async signUp(@Body({ validate: false }) user: User, @Res() newUser: User) {`